lib/ProductOpener/Export.pm

```diff
@@ -279,37 +279,7 @@ sub export_csv($args_ref) {
 				elsif ($group_id eq "images") {
 					if ($args_ref->{include_images_paths}) {
 						if (defined $product_ref->{images}) {
-
-							# First list the selected images
-							my %selected_images = ();
-							foreach my $imageid (sort keys %{$product_ref->{images}}) {
-
-								if ($imageid =~ /^(front|ingredients|nutrition|packaging|other)_(\w\w)$/) {
-
-									$selected_images{$product_ref->{images}{$imageid}{imgid}} = 1;
-									$populated_fields{"image_" . $imageid . "_file"} = sprintf("%08d", 10 * 1000 ) . "_" . $imageid;
-									# Also export the crop coordinates
-									foreach my $coord (qw(x1 x2 y1 y2 angle normalize white_magic coordinates_image_size)) {
-										if ((defined $product_ref->{images}{$imageid}{$coord})
-											and (($coord !~ /^(x|y)/) or ($product_ref->{images}{$imageid}{$coord} != -1))  # -1 is passed when the image is not cropped
-											) {
-												$populated_fields{"image_" . $imageid . "_" . $coord} = sprintf("%08d", 10 * 1000 ) . "_" . $imageid . "_" . $coord;
-										}
-									}
-								}
-							}
-
-							# Then list unselected images as other
-							my $other = 0;
-							foreach my $imageid (sort keys %{$product_ref->{images}}) {
-
-								if (($imageid =~ /^(\d+)$/) and (not defined $selected_images{$imageid})) {
-									$other++;
-									$populated_fields{"image_" . "other_" . $other . "_file"} = sprintf("%08d", 10 * 1000 ) . "_" . "other_" . $other;
-									# Keep the imgid for second loop on products
-									$other_images{$product_ref->{code} . "." . "other_" . $other} = { imgid => $imageid};
-								}
-							}
+							(%other_images, %populated_fields) = include_image_paths($product_ref, \%populated_fields, \%other_images);
```

Comment:
```suggestion
							include_image_paths($product_ref, \%populated_fields, \%other_images);
```
Now that it's working with references, `%populated_fields` and `%other_images` get modified in the subroutine. No need to return anything from the subroutine.

Reply:
Sounds good!